utils/debug/3dTilesDebug.js

```diff
@@ -56,6 +56,7 @@ export default function create3dTilesDebugUI(datDebugTool, view, layer) {
                     const geometry = new THREE.SphereGeometry(node.boundingVolume.sphere.radius, 32, 32);
                     const material = new THREE.MeshBasicMaterial({ wireframe: true });
                     helper = new THREE.Mesh(geometry, material);
+                    helper.position.copy(node.boundingVolume.sphere.center);
```

Comment:
For 3d Tiles, there's no example with bouding sphere helper?

Reply:
There's one, but it's hidden and a bit hard to access with the globe controller. Here is a screenshot : ![sphere](https://user-images.githubusercontent.com/1814191/35568598-d82e29b2-05c9-11e8-8e0f-a879848908df.jpg)

Reply:
yes, it is indeed very difficult to see, it should be made more accessible